tools/klee-stats/Makefile

```diff
@@ -20,30 +22,46 @@ include $(LEVEL)/Makefile.common
 # FIXME: Move this stuff (to "build" a script) into Makefile.rules.
 
 ToolBuildPath := $(ToolDir)/$(TOOLSCRIPTNAME)
+ConfigBuildPath := $(ToolDir)/$(TOOLCONFIGNAME)
+ConfigDirPath := $(ToolDir)/$(CONFIGDIRNAME)
 
-all-local:: $(ToolBuildPath)
+all-local:: $(ToolBuildPath) $(ConfigBuildPath) $(ConfigDirPath)
 
 $(ToolBuildPath): $(ToolDir)/.dir
+$(Tool2BuildPath): $(ToolDir)/.dir
 
 $(ToolBuildPath): $(PROJ_SRC_DIR)/$(TOOLSCRIPTNAME)
 	$(Echo) Copying $(BuildMode) script $(TOOLSCRIPTNAME)
 	$(Verb) $(CP) -f $(PROJ_SRC_DIR)/$(TOOLSCRIPTNAME) "$@"
 	$(Verb) chmod 0755 "$@"
 
+$(ConfigBuildPath): $(PROJ_SRC_DIR)/$(TOOLCONFIGNAME)
```

Comment:
The CMake build is missing the code to copy over the configuration file and version file.